src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -1124,36 +1128,32 @@ public void refreshUserAppsDictionary() {
                 Iterators.size( entitySetData.iterator() )
         );
 
-        Map<String, String> appsDict = new HashMap<>();
+        Set<String> systemUserApps = new HashSet<>();
```

Comment:
Same comment about `systemUserApps`. Please rename to `systemApps`. Also, I recommend matching the name with the instance field name

```
Set<String> systemUserPackageNames = new HashSet<>();
...
this.systemUserPackageNames.addAll( systemUserPackageNames );
```